sos/plugins/console.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from glob import glob
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class Console(Plugin, RedHatPlugin):
```

Comment:
@BryanQuigley @battlemidget any desire to have this in Ubuntu as well?

Reply:
Yes, please -- although I'll still +1 if it's not added.

Reply:
I'll add Ubuntu. Thanks.